### DISCUSSION TITLE
Arachne not showing/apearing

### DISCUSSION THREAD

**Discussion by JnakaLV** [supermerill/SuperSlicer#3300] [Q&A]
I tried to check new version with Arachne advantage and aim not able to find
Searching in wen that show are under Print setings -> Perimeters & Shell scrol down to see Arachne
But for me nothing
![unknown_2022 09 30-15 30](https://user-images.githubusercontent.com/111788789/193270650-9b59ced3-a392-4cdd-9feb-e3d8604567dc.png)

**Comment by neophyl** [comment]
You arent having this issue are you https://github.com/supermerill/SuperSlicer/issues/3293  ?
